### ISSUE TITLE
Word / characters count not reported / inaccessible

### ISSUE THREAD

**Issue by mmotyczynska** [ckeditor/ckeditor5#11849]
## 📝 Provide detailed reproduction steps (if any)

1.  Open [full-featured editor](https://ckeditor.com/docs/ckeditor5/latest/examples/builds-custom/full-featured-editor.html).
2.  Try to navigate to words/characters count section with a keyboard.
3.  Run voiceover for the editor, try to move focus to the words/characters count section (e.g. using CapsLock + arrow-right)

### ✔️ Expected result

Words/characters count section is accessible and easy to navigate.

### ❌ Actual result

**Keyboard**: it's not possible to navigate to this section.

**Voiceover**: it is possible to navigate to the section and that's great, but if we had a lot of contents including several tables inside the editor then it would take too much time to go through all of them down to the bottom just to check the current words/characters count. It should be much easier/quicker.

### ❓ Possible solution

_If you have ideas, you can list them here. Otherwise, you can delete this section._

## 📃 Other details

*   Browser: …
*   OS: …
*   First affected CKEditor version: …
*   Installed CKEditor plugins: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
We had a refinement call on this one. Here are some notes:

@Comandeer pointed out that CKE5 provides only API for word / character count. As such it's up to implementers to ensure that this information is accessible.

## Possible solutions

*   @Comandeer notes that one solution is to implement a hotkey to move the focus to bottom bar just like in [CKE4](https://ckeditor.com/docs/ckeditor4/latest/examples/standardpreset.html) (you can press <kbd>alt</kbd>+<kbd>f11</kbd>).
*   @mlewand we could consider using `aria-live` region with polite policy so that the user gets updated with the change.
    *   However the default implementation might imply a bit too often updates which would harm the feature.
*   Another option would be to make a dedicated hotkey to show the information about word count / character count information.
    *   We could either just read this data for screen readers (by placing it in some hidden but accessible for screen reader `aria-live` element, possibly with `assertive` value).
        *   The problem with this solution is that this would benefit only SR users.
    *   We could display word/char information in an `alert()`.
        *   This would be ugly :smile:
        *   But would do the job for all the users.
*   @mmotyczynska suggested that we could add a hotkey to focus the bottom status bar ([where the stats are printed](https://ckeditor.com/docs/ckeditor5/34.0.0/examples/builds-custom/full-featured-editor.html)).
    *   However the "bottom status bar" is not a part of our standard API - it's just part of custom code for the example - so this solution isn't applicable.
